docs/development/ceps/cep-001.rst

```diff
@@ -0,0 +1,243 @@
+.. _cep-001:
+
+
+**********************************
+CEP 1 - Purpose and guidelines
+**********************************
+
+* Status: draft
+* Discussion:  NA
+* Date accepted: NA
+* Last revised: Febuary 10, 2023
+* Author: Tomas Bylund
+* Created: Febuary 10, 2023
+
+Abstract
+========
+
+CEP stands for "Ctapipe Enhancment Proposal". This is CEP 1, which
+describes the purpose of writing CEPs as well as giving some guidelines
+on how CEPs are authored, discussed and reviewed.
+
+An CEP is a design document providing information to the community, or 
+describing a new feature for ctapipe or its processes or environment. 
+The CEP should provide a concise technical specification of the feature
+and a rationale for the feature.
+
+We intend CEPs to be the primary mechanisms for proposing major new features,
+for collecting community input on an issue, and for documenting the design
+decisions that have gone into Python. The CEP author is responsible for
+building consensus within the community and documenting dissenting opinions.
+
+Because the CEPs are maintained as text files in a versioned repository
+(indirectly since this wiki is versioned within github), their revision
+history is the historical record of the feature proposal
+
+
+What is a CEP?
+==============
+
+CTApipe Enhancment Proposals are short documents proposing a major
```

Comment:
```suggestion
ctapipe Enhancment Proposals are short documents proposing a major
```

Reply:
I like this sentence much better than the above introduction sentence that only mentions new features. This is a bit duplicated, so maybe just use the one here?